Sources/PointFree/Account/Account.swift

```diff
@@ -767,4 +767,9 @@ private struct AccountData {
   var isSubscriptionOwner: Bool {
     return self.currentUser.id == self.subscriptionOwner?.id
   }
+
+  var isTeamSubscription: Bool {
+    guard let id = stripeSubscription?.plan.id else { return false }
```

Comment:
can you use `self.stripeSubscription`? we like explicit `self`s.

Reply:
we like the distinction between local/global/instance vars more than being explicit about capture. and there's not a single class in the entire Point-Free code base so no risk of capture!

Reply:
ahh, got ya — thanks, and the beauty of not being tied to Cocoa/Cocoa Touch!